.circleci/config.yml

```diff
@@ -81,6 +84,7 @@ commands:
             <<# parameters.enable_native_custom_element_lifecycle >> ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE=1 <</ parameters.enable_native_custom_element_lifecycle >> \
             <<# parameters.enable_scoped_custom_element_registry >> ENABLE_SCOPED_CUSTOM_ELEMENT_REGISTRY=1 <</ parameters.enable_scoped_custom_element_registry >> \
             <<# parameters.disable_aria_reflection_polyfill >> DISABLE_ARIA_REFLECTION_POLYFILL=1 <</ parameters.disable_aria_reflection_polyfill >> \
+            <<# parameters.disable_synthetic_shadow_support_in_compiler >> DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER=1 <</ parameters.disable_synthetic_shadow_support_in_compiler >> \
```

Comment:
Verified manually that this Karma test works with the config passed in.

![Screen Shot 2022-12-15 at 5 33 58 PM](https://user-images.githubusercontent.com/283842/208001610-69d5d4a1-403d-43cf-9753-bb8044d2684c.png)

In the above screenshot, some of the functions still have their arguments, because they are scoped (`stylesheet.$scoped$ = true`).